open-api/rest-catalog-open-api.yaml

```diff
@@ -1500,6 +1500,9 @@ components:
           properties:
             schema:
               $ref: '#/components/schemas/Schema'
+            last-column-id:
+              type: integer
+              description: The highest assigned column ID for the table. This is used to ensure columns are always assigned an unused ID when evolving schemas.
```

Comment:
This isn't technically required in the OpenAPI spec. We send it in the Java client because we already have it. But there isn't really a requirement for it because the server side can find `max(id for id in schema)`

Reply:
Got it. It is also not required in the OpenAPI spec. I updated the Java side as well to make it optional. We can also omit it from the spec entirely, I don't have a strong opinion here.

Reply:
I'd say let's make it optional and document what Java does, but not force people to do the same.

Reply:
```suggestion
              description: The highest assigned column ID for the table. This is used to ensure columns are always assigned an unused ID when evolving schemas. When omitted, it will be computed on the server side.
```